SparkPost/2web2ui
# SA-2248: Deliverability metrics changes

### What Changed

- Changed industry benchmark modal button to "Apply Settings"
- Changed calculation for moved to inbox and moved to spam rates
- Added tooltips for deliverability metrics


### How To Test
- Use an account with d12y metrics enabled (appteam)
- Open metrics modal
- Verify d12y metrics have tooltip (and content is correct)
- Verify Accepted metrics removed from this d12y section
- Add "Inbox Folder Rate" as metric,
-  then above chart, click "Industry Benchmark"
- Verify text has changed to "Apply Settings"
